### PR TITLE
Converted Seattle Lidar example to a project

### PR DESCRIPTION
**Pull request by jlstevens**
Converts the [Seattle Lidar example on EarthML](https://earthml.holoviz.org/topics/Seattle_Lidar.html) to a project. Testing and running correctly locally. This project might be able to work on Travis as long as the intake download from S3 can happen fast enough. When the necessary data is in the cache, the notebook runs fairly quickly.

### CODE REVIEW

File: seattle_lidar/Seattle_Lidar.ipynb [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.0"
```
Comment (jbednar) [inline seattle_lidar/Seattle_Lidar.ipynb:456]: Did you mean for this metadata to be included?

File: seattle_lidar/anaconda-project.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - geoviews==1.6.6
+  - holoviews==1.12.7
+  - python-snappy=0.5.4
+  - intake==0.5.4
+  - intake-xarray==0.3.1
+  - rasterio==1.0.25
+  - dask==2.9.2
+  - s3fs==0.4.0
+  - pandas==0.25.3
+  - distributed=2.9.3
```
Comment (jbednar) [inline seattle_lidar/anaconda-project.yml:30]: What's the difference between = and ==? Presumably it should pick one or the other, preferably for all of the examples on this site? Same for spacing; is the whitespace in " <6"  significant?

File: seattle_lidar/anaconda-project.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    unix:    pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    env_spec: test
+
+env_specs:
+  default: {}
+  test:
+      packages:
+      - nbsmoke ==0.2.8
+      - pytest ==4.4.1
```
Comment (jbednar) [inline seattle_lidar/anaconda-project.yml:54]:
```suggestion
      - pytest ==4.4.1

```

File: seattle_lidar/Seattle_Lidar.ipynb (hunk 1 above)
Comment (jlstevens) [reply seattle_lidar/Seattle_Lidar.ipynb]: Fixed in d1e28bf

File: seattle_lidar/anaconda-project.yml (hunk 2 above)
Comment (jlstevens) [reply seattle_lidar/anaconda-project.yml]: Fixed in f9e7230 - I agree the formatting should be consistent.

**Comment by jlstevens** [comment]
Tests are green and the comments addressed. I need this merged to master to continue with the website builds. I'm merging now but happy to open another PR if further tweaks are needed.

**Pull request merged**